service-discovery/static-list/revapi.json

```diff
@@ -24,7 +24,24 @@
     "criticality" : "highlight",
     "minSeverity" : "POTENTIALLY_BREAKING",
     "minCriticality" : "documented",
-    "differences" : [ ]
+    "differences" : [
+      {
+        "ignore": true,
+        "code": "java.annotation.removed",
+        "old": "class io.smallrye.stork.servicediscovery.staticlist.StaticListServiceDiscoveryProvider",
+        "new": "class io.smallrye.stork.servicediscovery.staticlist.StaticListServiceDiscoveryProvider",
+        "annotation": "@io.smallrye.stork.api.config.ServiceDiscoveryAttribute(name = \"address-list\", description = \"a comma-separated list of addresses\")",
+        "justification": "By adding a second annotation, it switches to 'ServiceDiscoveryAttributes'. Should not impact users."
+      },
+      {
+        "ignore": true,
+        "code": "java.annotation.added",
+        "old": "class io.smallrye.stork.servicediscovery.staticlist.StaticListServiceDiscoveryProvider",
+        "new": "class io.smallrye.stork.servicediscovery.staticlist.StaticListServiceDiscoveryProvider",
+        "annotation": "@io.smallrye.stork.api.config.ServiceDiscoveryAttributes({@io.smallrye.stork.api.config.ServiceDiscoveryAttribute(name = \"address-list\", description = \"A comma-separated list of addresses (host:port). The default port is 80.\"), @io.smallrye.stork.api.config.ServiceDiscoveryAttribute(name = \"secure\", description = \"Whether the connection with the service should be encrypted with TLS. Default is false, except if the host:port uses the port is 443.\")})",
+        "justification": "By adding a second annotation, it switches to 'ServiceDiscoveryAttributes'. Should not impact users."
```

Comment:
That's the beauty of Java...
Before we had a single annotation, so it was: `ServiceDiscoveryAttribute`.
Now, we have 2 annotations, so Java introduces a container annotation `ServiceDiscoveryAttributes` (plural), which contains the 2 `ServiceDiscoveryAttribute` (backed in an array). 

And yes, it can be seen as a breaking change for programs looking at the `ServiceDiscoveryAttribute` annotations, which would not find anything anymore as they need to look at `ServiceDiscoveryAttributes` and gets the array of `ServiceDiscoveryAttribute`. No worries for us, we already do that work.

Reply:
ok. I see yes, but the `address-list` did existe for a while now, right? It's just that the revapi hadn't traced it when it wass added, no?

Reply:
`address-list` was the top level `@ ServiceDiscoveryAttribute`. Now it's in the nested array:

```
@ ServiceDiscoveryAttributes( {
   @ ServiceDiscoveryAttribute, // address-list
   @ ServiceDiscoveryAttribute // secure
})
```